public/app/features/explore/ExploreActions.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC, useEffect, useState } from 'react';
+import { useRegisterActions, useKBar, Action } from 'kbar';
+import { useDispatch, useSelector } from 'react-redux';
+import { StoreState, ExploreId } from 'app/types';
+import { splitOpen, splitClose } from './state/main';
+import { isSplit } from './state/selectors';
+import { runQueries } from './state/query';
+
+interface Props {
+  exploreId: ExploreId;
+}
+
+export const ExploreActions: FC<Props> = ({ exploreId }: Props) => {
+  const [actions, setActions] = useState<Action[]>([]);
+  const { query } = useKBar();
+  const dispatch = useDispatch();
+  const { splitted } = useSelector((state: StoreState) => {
+    const splitted = isSplit(state);
+    return {
+      splitted,
+    };
+  });
+
+  useEffect(() => {
+    const actionsArr: Action[] = [
+      {
+        id: 'explore/run-query',
+        name: 'Run Query',
+        keywords: 'query',
+        perform: () => {
+          dispatch(runQueries(exploreId));
+        },
+        section: 'Explore',
```

Comment:
more of a nit than anything else.
I think local actions (in this case all these explore actions) should be positioned at the top, it feels more natural to have actions contextual to what i am doing higher in the action list. i think you can achieve that by prioritizin the section:

```suggestion
        section: {
          name: 'Explore',
          priority: Priority.HIGH,
        },
```